spec/takeaway_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:order_klass) {
+    double :order_klass, new: menu_klass, calculate_quantities: order,
+    calculate_cost: order, calculate: order
+  }
+  subject(:takeaway) { described_class.new(order_klass) }
+  let(:order) {
+    double :order, inject: [["Dish 1" => 5.00]],
+    length: 4, each: nil, calculate_cost: 5
+  }
+  let(:quantity) { double :quantity => 4}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)